### PR TITLE
Translation to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by afmachado**

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <string name="dialog_reflector_set">Refletor Religado.</string>
+    <string name="dialog_plugboard_set">Definir configurações do Quadro de Conexões.</string>
+    <string name="dialog_passphrase_set">Gerar configurações a partir de senha %1$s.</string>
+    <string name="dialog_passphrase_was_coded_state">Configurações restauradas do estado codificado.</string>
+    <string name="dialog_abort">Sem alterações.</string>
+    <string name="message_reset">Restaurar o Enigma.</string>
+    <string name="message_random">Enigma definido para configurações aleatórias.</string>
+    <string name="message_clipboard">Copiado para a área de transferência</string>
+
+    <string translatable="false" name="button_plug_title">%1$s:%2$s</string>
```
Comment (vanitasvitae) [inline app/src/main/res/values-pt-rBR/strings.xml]: Please remove untranslatable values (this one and all below)

**Review by vanitasvitae — CHANGES_REQUESTED** [review]
There are untranslatable strings, which can be removed.

**Comment by vanitasvitae** [comment]
Hi @afmachado,
Thank you very much for your contribution!
Please remove untranslatable strings, then I will be happy to merge :)

**Comment by afmachado** [comment]
updated

**Comment by vanitasvitae** [comment]
Great! Thank you!

**Pull request merged**